src/common/fees/makeFees.ts

```diff
@@ -110,11 +111,17 @@ export const makeFees = async (config: MakeFeesConfig): Promise<Fees> => {
 const fetchCachedFees = async (
   memlet: Memlet,
   currencyInfo: EngineCurrencyInfo
-): Promise<SimpleFeeSettings> =>
-  await memlet
+): Promise<undefined | SimpleFeeSettings> => {
+  const data = await memlet
     .getJson(FEES_PATH)
     // Return the simple fees settings from currency info by default
     .catch(() => currencyInfo.simpleFeeSettings)
+  try {
+    return asSimpleFeeSettingsCleaner(data)
+  } catch (_) {
+    return
+  }
+}
```

Comment:
This is an opportunity to use `asMaybe`

Reply:
Fixed in: https://github.com/EdgeApp/edge-currency-plugins/pull/151/commits/38e026c454894fc1740a4ffa7838fa37f027209a